RadioMark/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react'
+import { Motion, spring } from 'react-motion'
+import compose from '../lib/compose'
+import themeable from '../decorators/themeable'
+
+const grayLines = '#cbcbcd'
+const klarnaBlue = '#0074c8'
+const white = 'white'
+
+function RadioMark ({ customize, checked, disabled, onClick, lowFPS }) {
+  const backgroundColor = customize ? customize.backgroundColor : klarnaBlue
+  const bulletColor = customize ? customize.bulletColor : white
+  const scale = checked ? 1 : 3
```

Comment:
I think these magic numbers should have names, like `SCALE_FACTOR_[CHECKED|UNCHECKED]` or something.